doc/recipe/overview.rst

```diff
@@ -181,6 +181,11 @@ arguments):
 
 .. _Diagnostics:
 
+Preprocessor operations will be applied using the default order
```

Comment:
Maybe it would be nicer to add this text to the section above? https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/overview.html#recipe-section-preprocessors

Reply:
Sounds good. 

One question however. Is there a difference between:
"adding `custom: true` to the preprocessor definition"
and "specified using the custom_order preprocessor function." (as stated now above?)

Reply:
Thanks for pointing that out, I didn't know that text was there: it is wrong and should be removed, there is no `custom_order` preprocessor function.

Reply:
To be more specific, what I meant by adding it to the section above, is moving the ``.. _Diagnostics:`` label of the diagnostics section below this new text.

Reply:
Sorry, I was confused about the exact name, indeed the only working solution is `custom_order: true`, just `custom: true` doesn't do anything. The wording above is a bit unfornate though, because it isn't really a preprocessor function, but just a way to change the order in which the preprocessor functions are executed.

Reply:
Just removed previous text referencing custom_order as a function. Since this is a couple lines long seems to work well here instead of as a bullet point above.